scss/_utilities.scss

```diff
@@ -77,7 +77,16 @@ $utilities: map-merge(
     "border-color": (
       property: border-color,
       class: border,
-      values: map-merge($theme-colors, ("white": $white))
+      values: map-merge(
+        $theme-colors,
+        (
+          "white": $white,
+          "gray-200": $gray-200,
```

Comment:
The default border color is 300, weird not to have it here:
https://github.com/twbs/bootstrap/blob/8abccff7b3939334c61286df448d0f730da85325/scss/_variables.scss#L318